openfe/tests/protocols/conftest.py

```diff
@@ -10,17 +10,23 @@
 import pooch
 
 
+@pytest.fixture(scope='session')
+def charged_benzene(benzene_modifications):
+    benzene_offmol = benzene_modifications['benzene'].to_openff()
+    benzene_offmol.assign_partial_charges(partial_charge_method='gasteiger')
```

Comment:
Is it pre-charging the molecule that does the speed-up here? If so, can we just include the charges hard-coded in the file, so that we  1) don't need to generate charges at all and 2) remove any reproducibility issues stemming from the partial charge method?